packages/docs/getting-started/using-metamask-with-celo/README.md

```diff
@@ -1,32 +1,44 @@
+
+## **Private Key Import**
+
+Celo and Ethereum use different derivation paths for generating seed phrases. Because MetaMask does not let you specify a derivation path to use:
 
-* Celo and Ethereum use different derivation paths for generating seed phrases. Because MetaMask does not let you specify a derivation path to use:
   * You can't import an existing Celo account into the MetaMask wallet using its seed phrase, as you'd get the Ethereum version of it. Instead, you have to import it using the associated private key.
-  * Similarly, if you want to import the Celo account you made on MetaMask to a different Celo wallet \(e.g. [Valora](https://valoraapp.com/)\) you'd have to import it using the private key itself, NOT the seed phrase that MetaMask gives you.
-  * See these guides if you accidentally sent [ETH to CELO addresses](https://docs.celo.org/celo-owner-guide/celo-recovery) or [CELO to ETH addresses.](https://docs.celo.org/celo-owner-guide/eth-recovery)\*\*\*\*
+  * If you want to import the Celo account you made on MetaMask to a different Celo wallet (e.g. [Valora](https://valoraapp.com/)) you'd have to import it using the private key itself, NOT the seed phrase that MetaMask gives you.
+  * See these guides if you accidentally sent [ETH to CELO addresses](https://docs.celo.org/celo-owner-guide/celo-recovery) or [CELO to ETH addresses.](https://docs.celo.org/celo-owner-guide/eth-recovery)
 
-### Gas Fee Currency
+## **Gas Fees Require CELO**
 
-* While gas on Celo can usually be paid in [many different currencies](https://docs.celo.org/celo-codebase/protocol/transactions/erc20-transaction-fees), when using Metamask, gas fees will automatically be paid in CELO. This is because MetaMask will be using the [Ethereum-compatible Celo transaction format](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0035.md), which doesn't include the `feecurrency` field.
+While gas on Celo can usually be paid in [many different currencies](https://docs.celo.org/celo-codebase/protocol/transactions/erc20-transaction-fees), when using MetaMask, gas fees will automatically be paid in CELO. This is because MetaMask will be using the [Ethereum-compatible Celo transaction format](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0035.md), which doesn't include the `feecurrency` field.
 
-### Logo
+## **Incorrect Logo**
 
-* The MetaMask UI might show the ETH logo in places where it's meant to show the CELO logo or no logo at all.
+In some cases, the MetaMask UI may display the Ethereum logo in places where it should display a CELO logo or no logo at all.
+
+{% hint style="warning" %}
+MetaMask is primarily used for interacting with the Ethereum blockchain and does not natively support Celo compatibility. Celo recommends only sophisticated users and developers use it. Alternatively, you may choose a Celo native wallet [here](https://docs.celo.org/getting-started/wallets).
```

Comment:
Are we still recommending that only sophisticated users and developers use Metamask, or are we now recommending it for everyone (considering pool together only supports metamask)?

Reply:
This should be moved to bottom.

We want to diminish severity of the warning and let people choose how they wish to interact. PT needs MM, so yes recommending it as a viable option versus warning to stay away.

Reply:
Kept at the bottom with lighter language than before